content/testing-yaml/testing.md

```diff
@@ -7,6 +7,8 @@ aliases:
 ---
 
 Test scripts are added under `scripts` in the [overall architecture](../getting-started/yaml#template), before the build commands.
+You can display test results visually in the build overview if you use expanded form of the script in codemagic.yaml.
+Just include the `test_report` field with a glob pattern matching the test result file location. Supported test report formats are Junit and Flutter `--machine` report.
```

Comment:
```suggestion
Just include the `test_report` field with a glob pattern matching the test result file location. Supported test report formats are [Junit XML](https://llg.cubic.org/docs/junit/) and Flutter `--machine` report.
```

Reply:
Perhaps it is worth noting that Junit XML files need to be with `.xml` or `.junit` suffix and Flutter `--machine` reports are expected to have `.json` suffix?

Reply:
fixed in 2e33bafc3592fa7408d75ac9459306eb97d56a8d